inventory/byo/hosts.localhost

```diff
@@ -0,0 +1,21 @@
+#bare minimum hostfile
+
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ansible_ssh_user=root
+debug_level=2
+openshift_deployment_type=origin
+openshift_release=3.7
+
+[masters]
+localhost
+
+[etcd]
+localhost
+
+[nodes]
+localhost openshift_schedulable=true openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
```

Comment:
I don't think registry and router will be scheduable without region: infra, unless you change their selectors.

Reply:
I believe labels can take a list.